packages/terra-form-validation/docs/README.md

```diff
@@ -0,0 +1,11 @@
+# Form Validation
+
+To write form validations for your component, Terra recommends using [react-final-form](https://github.com/final-form/react-final-form). This package contains several different hooks and functionality for performing validations at various levels which include:
+
+- Validating onChange
+- Validating onSubmit
+- Validation synchronously (Such as for unit user name implementations)
```

Comment:
This may be a dumb question but what do you mean by unit user name implementations?

Reply:
Was `unit` a typo that meant to read `unique`?

Reply:
It is supposed to be unique. I'll get it updated.

Reply:
Should this be `Synchronous Validation`?